internal/cocoasdk/nsmethodsignature_darwin.go

```diff
@@ -0,0 +1,20 @@
+package cocoasdk
+
+import "github.com/ebitengine/purego/objc"
+
+var class_NSMethodSignature = objc.GetClass("NSMethodSignature\x00")
+
+var (
+	sel_instanceMethodSignatureForSelector = objc.RegisterName("instanceMethodSignatureForSelector:\x00")
+	sel_signatureWithObjCTypes             = objc.RegisterName("signatureWithObjCTypes:\x00")
+)
+
+type NSMethodSignature objc.ID
+
+func InstanceMethodSignatureForSelector(self objc.ID, _cmd objc.SEL) NSMethodSignature {
```

Comment:
What about `cmd` instead of `_cmd`?

Reply:
I did _cmd bc that is how it is done in Objc but if you prefer just cmd then I can do that

Reply:
> I did _cmd bc that is how it is done in Objc

Oh really? Could you show me a link for the explanation?

Reply:
The reason is likely because they didn't want to take up the identifier `cmd` from use in a method. This [Apple Doc](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ObjCRuntimeGuide/Articles/ocrtHowMessagingWorks.html) says "The first two arguments passed to the procedure are the receiving object (self) and the method selector (_cmd). These arguments are hidden in method syntax but must be made explicit when the method is called as a function." Although it doesn't say _why_ it is this way.

Reply:
OK I'm convinced.